teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(_name, age, specialization)
+    super('Unknown', age)
```

Comment:
- [ ] Please kindly declare the parameters inherited from the `Person.rb` appropriaely as you did in `Person.rb` file. The `super()` function is used to call methods and constructors of the parent class, kindly ensure that the parameters are written correctly. 👍 **Note: parent_permission=true is missing from the constructor as well**